backend/routes/student.ts

```diff
@@ -193,20 +193,54 @@ async function createStudentSuggestion(
         return Promise.reject(errors.cookInvalidID());
     }
 
+    const osocYear = await ormOs.getLatestOsoc();
+
+    if (osocYear == null) {
+        return Promise.reject(errors.cookNoDataError());
+    }
+
+    console.log(await ormJo.getStudentEvaluationsTemp(student.student_id));
```

Comment:
I feel like this would clutter the console?

Reply:
Yeah, It's been removed, sorry